### PR TITLE
Sonarqube cleanup

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: implementation/common/src/main/java/org/inaetics/dronessimulator/common/D3PolarCoordinate.java [hunk 1]
```diff
@@ -27,11 +28,14 @@ public D3PolarCoordinate() {
...
      * Instantiates a new three-dimensional polar coordinate with the given coordinates.
-     * @param angle1_x_y The angle between the x and y axis.
-     * @param angle2_x_z The angle between the x and z axis.
-     * @param length The distance to the coordinate.
+     * @param angle1_x_y_ The angle between the x and y axis.
+     * @param angle2_x_z_ The angle between the x and z axis.
+     * @param length_ The distance to the coordinate.
      */
-    public D3PolarCoordinate(double angle1_x_y, double angle2_x_z, double length) {
+    public D3PolarCoordinate(double angle1_x_y_, double angle2_x_z_, double length_) {
```
Comment (jjkester) [inline implementation/common/src/main/java/org/inaetics/dronessimulator/common/D3PolarCoordinate.java:35]: Why this change?

**Review by jjkester — APPROVED** [review]

File: implementation/common/src/main/java/org/inaetics/dronessimulator/common/D3PolarCoordinate.java (hunk 1 above)
Comment (ghost) [reply implementation/common/src/main/java/org/inaetics/dronessimulator/common/D3PolarCoordinate.java:35]: naming duplication. Was changing the parameter length with another value and SonarQube saw that was a bug. So I created a variable length with `length = length_`. To avoid changing the names later on I changed the parameter name instead of the variable name in the body.
